Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,32 @@
 
```

Comment:
These are all the correct answers!

One thing I'll mention is that when you provide an explanation for the run-time complexity it should be a little more thorough. For example:

`O(n) because as the input increases, the run time will increase at the same rate`

My follow-up question would be "Why in this example does the runtime increase at the same rate?" 

It's not so obvious why when you're given this to start with:

```
while (a < n * n * n):
      a = a + n * n
```

Something like: "While the while-loop runs at `O(n^3)` for `a < n^3`, `a` increases at a rate `O(n^2)` for each iteration. Therefore, O(n^3 - n^2) = O(n)